app/src/main/kotlin/cz/covid19cz/erouska/ui/about/AboutVM.kt

```diff
@@ -11,4 +12,7 @@ class AboutVM @ViewModelInject constructor() : BaseVM() {
         publish(UrlEvent(AppConfig.conditionsOfUseUrl))
     }
 
+    fun versionClicked(){
```

Comment:
Dal bych to na long press, ne na klik. Ať se to neděje náhodně.

Reply:
fixed